lamp/tasks/create_certificate.yml

```diff
@@ -26,10 +26,10 @@
 # h/t to https://serialized.net/2013/04/simply-generating-self-signed-ssl-certs-with-ansible/
 - name: create self-signed SSL cert
   become: yes
-  command: openssl req -new -nodes -x509 -subj "/C=US/ST=Virginia/L=Richmond/O=Foster Made/CN=Local Dev" -days 3650 -keyout /etc/ssl/private/{{ hostname }}.key -out /etc/ssl/certs/{{ hostname }}.crt -extensions v3_ca
+  command: openssl req -new -nodes -x509 -subj "/C=US/ST=Virginia/L=Richmond/O=Foster Made/CN=*{{ ssl_cert }}" -days 3650 -keyout /etc/ssl/private/{{ ssl_cert_name }}.key -out /etc/ssl/certs/{{ ssl_cert_name }}.crt -extensions v3_ca
```

Comment:
The use of CN to identify any hostname value has been deprecated, and at least Chrome and Firefox no longer even look there for a hostname — see https://groups.google.com/a/chromium.org/forum/#!msg/security-dev/IGT2fLJrAeo/csf_1Rh1AwAJ. That was the general motivation to just use a generic string value here, as Let’s Encrypt does with their certs (using “Let's Encrypt Authority X3” as the CN value). I think we were only using the hostname as the key and cert name out of habit. But with the possibility of multiple hostnames, we can use something generic, hard-coded and as simple as ssl.crt. Then we can get rid of the `ssl_cert_name` var completely.